src/services/pathCompletions.ts

```diff
@@ -176,6 +177,15 @@ namespace ts.Completions.PathCompletions {
             result = [];
         }
 
+        if (compilerOptions.moduleResolution === ts.ModuleResolutionKind.NodeJs) {
+            forEachAncestorDirectory(currentDirectory, ancestor => {

```

Comment:
This shouldnt be from currentDirectory but from getDirectoryPath(scriptPath) isnt it ? (See https://github.com/Microsoft/TypeScript/blob/2ea723f3156b0e224d438330951e2bf1edf8c065/src/compiler/moduleNameResolver.ts#L751)